packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -262,17 +265,22 @@ export class ApexTestGroupNode extends TestNode {
 
   public updatePassFailLabel() {
     this.passing = 0;
+    this.failing = 0;
+    this.skipping = 0;
```

Comment:
What is the purpose of adding `this.skipping` if we never check for the outcome being 'Skip' and increment the count ? How can I test for this scenario ?

Reply:
Ah that was an oversight, I meant to increment `test.skipping` as well. I was wondering if we want to mimic the CLI's behavior here as well: if a test is run individually, it will give the user a message if the test was changed in the org and not locally. If a test is run as part of the whole test class, it does not give the user any message that a specific test name was altered. Instead, it shows that all tests were run. This leads to the status of the local test (that was not renamed) to display as `Outcome: Not Run`.